src/app/dataset-view/additional-components/data-component/data-component.ts

```diff
@@ -27,14 +27,18 @@ import { sqlEditorOptions } from "src/app/dataset-block/metadata-block/component
 })
 export class DataComponent extends BaseComponent implements OnInit {
```

Comment:
Serious bug right now: if I modify the SQL query, I expect the data presentation to be restarted. Instead it is now appended to existing data.
I.e.:
```
select
  *
from 'account.tokens.portfolio'
order by block_number asc
```

then change `asc` => `desc`

Reply:
I like the idea of saving state and comparing queries, but It needs more significant changes to this process
For now, I picked a simple solution, I added a method to reset limits.